lib/backend/k8s/k8s_test.go

```diff
@@ -320,6 +320,41 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 		go cb.ProcessUpdates()
 	})
 
+	AfterEach(func() {
+		// Clean up all resources.
+		err := c.Clean()
+		Expect(err).NotTo(HaveOccurred())
+
+		// Clean up any k8s network policy left over by test.
+		nps := networkingv1.NetworkPolicyList{}
+		err := c.clientSet.NetworkingV1().RESTClient().
+			Get().
+			Resource("networkpolicies").
+			Timeout(10 * time.Second).
+			Do().Into(&nps)
+		Expect(err).NotTo(HaveOccurred())
+
+		for _, np := range nps.Items {
```

Comment:
could we not just delete the namespace to get rid of all these things? I guess we can't delete the default namespace, but maybe if we create a namespace for every test (kinda like upstream k8s NetworkPolicy e2es) and just create the namespace in `BeforeEach` and delete it in `AfterEach`? Maybe it's too much work or we need to make sure things work in the default namespace, in which case this is fine.

Reply:
I suppose we could, but that's extra work for questionable gain. I think we should just clean up the things we know we may have created for now.